docs/en/operation/Metastore.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: global
+title: Metastore Management
+nickname: Metastore Management
+group: Operations
+priority: 0
+---
+
+* Table of Contents
+{:toc}
+
+Alluxio stores most of its metadata on the master node. The metadata includes the
+filesystem tree, file permissions, and block locations. Alluxio provides two ways
+to store the metadata: (1) with an on-disk, RocksDB-based metastore or (2) with an 
+on-heap metastore.
+
+## RocksDB Metastore
```

Comment:
this should also specify how to enable rocks with `alluxio.master.metastore`

Reply:
done